src/index.html

```diff
@@ -6,7 +6,42 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+
+  <body class="calendar calendar--start-mon">
+
+    <div class="calendar__month calendar__month--length-31">
```

Comment:
```suggestion
    <div class="calendar calendar--month-length--31 calendar--start-day--mon">
```
Add all modifiers here. Btw, start day modifier doesn't work correctly